frontend/packages/shared/src/api/paths.js

```diff
@@ -16,7 +16,11 @@ export const serviceConfigPath = (org, app) => `${basePath}/${org}/${app}/config
 
 // Datamodel
 export const createDatamodelPath = (org, app) => `${basePath}/${org}/${app}/datamodels/new`; // Post
-export const datamodelPath = (org, app, modelPath, saveOnly = false) => `${basePath}/${org}/${app}/datamodels/datamodel?${s({ modelPath, saveOnly })}`; // Get, Put, Delete
+export const datamodelPath = (org, app, modelPath, saveOnly = false) =>
```

Comment:
Are the formatting changes in this file supposed to be done?

Reply:
Hm. maybe IDE is reading the wrong config. 🤔 Let me investigate it.